tests/test_utils.py

```diff
@@ -148,3 +148,124 @@ def get_division_graphs():
     mapper = [("1_0", "1_0"), ("1_1", "1_1"), ("2_4", "2_4"), ("3_4", "3_4")]
 
     return G1, G2, mapper
+
+
+def get_gap_close_graphs():
+    """
+    G1
+                                      3_5 -- 3_6 -- -- -- 5_10
+    1_0 -- 1_1 -- -- -- 2_3 -- 2_4 -<
+                                      4_5 -- 4_6
+    G2
+                                      2_5 -- 2_6 -- -- -- 4_10
+    1_0 -- 1_1 -- 1_2 -- 1_3 -- 1_4 -<
+                                      3_5 -- 3_6
```

Comment:
@cmalinmayor I thought we had validation in the `TrackingGraph` about edges only spanning one frame, but these graphs don't seem to be causing errors and I can't find this validation in the `TrackingGraph`. Am I misremembering or did we drop it at some point?

Reply:
I was running into the `TrackingGraph` validation wanting 't' for every `node_id`, which stemmed from my data having gaps.

Reply:
> @cmalinmayor I thought we had validation in the `TrackingGraph` about edges only spanning one frame, but these graphs don't seem to be causing errors and I can't find this validation in the `TrackingGraph`. Am I misremembering or did we drop it at some point?

@msschwartz21 I also thought this was in the validation but I think we actually didn't add it, perhaps on accident...

Reply:
> I was running into the `TrackingGraph` validation wanting 't' for every `node_id`, which stemmed from my data having gaps.

@lupinthief In my current understanding of implementing tracks with gaps, we still want all nodes to have a time associated. The gaps in time are covered by the edges that skip over frames, but still connect nodes that have times (e.g. an edge from a node in time `t` to a node in time `t+n`). Can that cover your use case as well?